crates/sui-json-rpc/src/read_api.rs

```diff
@@ -230,7 +230,7 @@ impl RpcFullNodeReadApiServer for FullNodeApi {
         let (txn_data, txn_digest) = get_transaction_data_and_digest(tx_bytes)?;
         Ok(self
             .state
-            .dev_inspect_transaction(txn_data, txn_digest)
+            .dev_inspect_transaction(txn_data, txn_digest, 0)
```

Comment:
@666lcz what should I do here?

Reply:
Does it ever make sense for people the call dev inspect with an epoch number other than the current epoch number? 

If not, I think we can autofill the epoch number here(instead of making epoch as a parameter in the RPC API) by fetching the `0x5` object.

Even if the answer is yes, I think in most cases developers just wanted to use the current epoch number, so I would add an optional parameter to `dev_inspect_transaction(&self, tx_bytes: Base64)` -> `dev_inspect_transaction(&self, tx_bytes: Base64, epoch: Option<EpochId>)`

Reply:
> If not, I think we can autofill the epoch number here(instead of making epoch as a parameter in the RPC API) by fetching the 0x5 object.

I like the solution of fetching the state object to auto fill it. And the optional parameter makes sense I think. Thanks!